mesh/pmesh.hpp

```diff
@@ -84,6 +84,10 @@ class ParMesh : public Mesh
    mutable long glob_elem_offset, glob_offset_sequence;
    void ComputeGlobalElementOffset() const;
 
+   // Should Print() add the parallel interface as boundary (typically used for
+   // visualization purposes)?
```

Comment:
Never write documentation with a question.

```// Enable Print() to add the parallel interface as boundary (typically used for visualization purposes)```

Reply:
I am not sure I agree that this is more clear, but I am OK with changing it